addons/account/views/account_move_views.xml

```diff
@@ -623,43 +620,31 @@
             <field name="model">account.move</field>
             <field name="arch" type="xml">
                 <kanban class="o_kanban_mobile" sample="1" js_class="account_documents_kanban">
-                    <field name="move_type" invisible="1"/>
-                    <field name="journal_id"/>
-                    <field name="partner_id"/>
-                    <field name="ref"/>
-                    <field name="date"/>
-                    <field name="state"/>
+                    <field name="currency_id" invisible="1" readonly="state in ['cancel', 'posted']"/>
```

Comment:
readonly? Are you sure it is necessary?

Reply:
readonly is not necessary. I missed checking that, it was added earlier.